api_spec.yaml

```diff
@@ -1231,6 +1231,211 @@ paths:
                                         type: array
                                         items:
                                             type: string
+    /dashboard/api/tenants/list-with-user-count:
+        get:
+            tags:
+                - Multitenancy
+            summary: Get all tenants created in the core with the number of users for each tenant
+            parameters:
+                - name: authorization
+                  in: header
```

Comment:
See if you want to include user count on an optional basis, considering user count will be extra overhead on the API

Reply:
Removed the API and added an additional query param called `withUserCount` which would return the the user counts only when true to avoid the extra overhead